cmd/kubeadm/app/apis/kubeadm/v1beta2/types.go

```diff
@@ -228,7 +229,7 @@ type Networking struct {
 type BootstrapToken struct {
 	// Token is used for establishing bidirectional trust between nodes and control-planes.
 	// Used for joining nodes in the cluster.
-	Token *BootstrapTokenString `json:"token"`
+	Token *bootstraptokenutil.BootstrapTokenString `json:"token"`
```

Comment:
this is a breaking change, the bootstraptokenutil.BootstrapTokenString struct does not define json tags, so switching the serialized kubeadm type to use that struct would change serialization